app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  enum role: { client: 1, admin: 2, }
```

Comment:
You are missing `role` in your migration, but I see it in your `schema.rb` file - did you forget to push migration changes?

And something to consider, not a requirement:
```ruby
enum role {
  client: "client",
  admin: "admin"
}
```
This way in the database you would have more descriptive values than just `1` and `2`.

Reply:
@rafalrothenberger I don't have migration, because role is only an attribute and for now I have it as an integer.

Reply:
But there is no migration creating it - you shouldn't create tables and columns directly in the database, everything should be created through migrations.

Reply:
@rafalrothenberger Thank you, I've fixed it :)